proj3_chess/chess/coordinates.py

```diff
@@ -0,0 +1,47 @@
+from pydantic import BaseModel
+
+
+class MoveVector(BaseModel):
+    x: int
+    y: int
+
+    def __mul__(self, vector: "MoveVector"):
+        return MoveVector(x=self.x * vector.x, y=self.y * vector.y)
+
+
+MV = MoveVector
+
+
+def convert_num_to_str(num):
```

Comment:
I am so confused by this method in so many ways

why 25?? why 97?? print "hi"??? this is called `convert_num_to_str` as if this is the obvious way a number should be converted to a string what????

I understand from `Spot` that this somehow print the coordinates better but could not be bothered to understand how. THis function should not exist

Reply:
OK
There's the letter side of the board right? So if you can represent 26 lines using a si gle lettter (hence the 25, because we start cpunting from 0). Using ASCII seemed like the easiest way to convert a number into a letter, and 97 is where letters start in the chart. 
Hope this explains